aiohttp/web_response.py

```diff
@@ -108,6 +108,7 @@ def __init__(
         self._keep_alive = None  # type: Optional[bool]
         self._chunked = False
         self._compression = False
+        self._compression_strategy = None  # type: Optional[int]
```

Comment:
```suggestion
        self._compression_strategy: Optional[int] = None
```

Comments for types are from legacy code, let's not add any more.